app/src/main/res/layout/activity_database.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
On all this file Cirrus CI warns us with some messages like: "Missing `autofillHints` attribute" or "Hardcoded string "Get", should use `@string` resource". I think those warnings are founded and could be fixed =D